src/scss/cdb-utilities/helpers.scss

```diff
@@ -36,6 +37,17 @@
 .u-bSpace--m {
   margin-bottom: 8px;
 }
+.u-textSecondary {
+  color: $cTypo2;
+}
+.u-textSubtitle {
```

Comment:
In my opinion we should rely names over element types. We could follow name variables, so:

```css
.u-textPrimary {}
.u-textSecondary {}
.u-textTertiary {}
.u-textQuaternary {}
```

cc @javierarce